### PR TITLE
Added docker

### PR DESCRIPTION
**Pull request by Stelmaschyk**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+MYSQLDB_PASSWORD=1234
+MYSQLDB_DATABASE=books
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+JWT_SECRET=AbstractAutowireCapableBeanFactory.createBeanInstance
+JWT_EXPIRATION=3000000000000
+
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```
Comment (liliia-ponomarenko) [inline .env]: Don’t push the `.env` file with sensitive data. better create .env.templete.
Let’s add .env to .gitignore

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - 8081:8080
+      - 5005:5005
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url": "jdbc:mysql://mysql:3306/books",
+        "spring.datasource.username": "${MYSQLDB_USER}",
+        "spring.datasource.password": "${MYSQLDB_PASSWORD}",
+        "spring.jpa.hibernate.ddl-auto": "validate"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (liliia-ponomarenko) [inline docker-compose.yml]:
```suggestion
      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:${DEBUG_PORT}"
```

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! But let’s fix the issue with .env ;)

File: docker-compose.yml (hunk 2 above)
Comment (Elena-Bruyako) [reply docker-compose.yml]: not fixed

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Builder stage
```
Comment (Elena-Bruyako) [inline Dockerfile:1]: add `.env.sample` or `.env.template` file with the same properties as in .env file but without values and push it

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

File: Dockerfile (hunk 3 above)
Comment (Stelmaschyk) [reply Dockerfile:1]: I have seen mark about link on debug port, and I fixed in IDE. I don't understand why it pushed to git.

**Review by Elena-Bruyako — APPROVED** [review]
well done

**Pull request merged**